cmake/Dependencies.cmake

```diff
@@ -1270,6 +1270,21 @@ endif()
 
 # ---[ HIP
 if(USE_ROCM)
+  # This prevents linking in the libtinfo from /opt/conda/lib which conflicts with ROCm libtinfo.
+  # Currently only active for Ubuntu 20.04 and greater versions.
+  if(UNIX AND EXISTS "/etc/os-release")
```

Comment:
Didn't know that [`LINUX`](https://cmake.org/cmake/help/latest/variable/LINUX.html) environment variable has only been added in cmake-3.25, though in PyTorch CMake file it has long been defined (as UNIX check would return true even for CygWin)